stories/Atom/Input-group/input_group.scss

```diff
@@ -0,0 +1,156 @@
+@import "../../assets/scss/mixins";
+@import "../../assets/scss/breakpoints";
+@import "../../assets/scss/variables";
+
+.label__caption,
+.help__caption,
+.error__caption {
+  @extend %paragragh-font-20;
+}
+.help__caption {
+  font-size: 14px;
+}
+.error__caption {
+  font-size: 16px;
+  color: $color-red;
+}
+
+.input-group {
+  .field__image {
+    display: inline-block;
+    position: relative;
+    width: 100%;
+    &.search {
+      &:before {
+        @extend %field__image;
+        content: "";
+        background-image: url(#{$img-path-Iconback}/Search_black.svg);
+      }
+      &.disabledstate {
+        &:before {
+          content: "";
+          background-image: url(#{$img-path-Iconback}/Searchgray.svg);
+        }
+      }
+    }
+  }
+  .field__setdate {
+    position: relative;
+  }
+  .input-type {
+    margin: 8px 0px;
+    border: 2px solid $color-black;
+    font-size: 16px;
+    width: 100%;
+    height: 46px;
+    outline: none;
+    &::placeholder {
+      color: $color-gray-500;
+    }
+    @include devicebreak(small) {
+      width: 478px;
```

Comment:
@pallavikadam-srijan Don't fix the width of the fields as if we can use these fields in any width of container so fields should be flexible